Add developer portal documentation (#589)
wso2/docs-choreo-dev: en/docs/developer-portal/test-api.md

> # Test API

Reviewer: Should be something like "Test an API" or alternately "Testing an API" depending on the naming convention we use for titles.

>  Before using your API in production, you can test it using the integrated OpenAPI console in Choreo. Once the API has is deployed in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test it's functionality by providing values for the required parameters. 

Reviewer: ```suggestion
Before using your API in production, you can test it using the integrated OpenAPI console in Choreo. Once the API is deployed in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test its functionality by providing values for the required parameters. 
```
